### PR TITLE
IGNITE-22423 Don't use kill -9 for ignite3db

### PR DESCRIPTION
**Pull request by Pochatkin**
https://issues.apache.org/jira/browse/IGNITE-22423

### CODE REVIEW

File: packaging/db/zip/linux/ignite3db [hunk 1]
```diff
@@ -59,7 +59,19 @@ start() {
 }
 
 stop() {
-  kill -9 "$(cat ${IGNITE_HOME}/pid)"
+  kill -15 "$(cat ${IGNITE_HOME}/pid)"
```
Comment (Lozmanov) [inline packaging/db/zip/linux/ignite3db:62]: move "$(cat ${IGNITE_HOME}/pid)" to variable and reuse it below

File: packaging/db/zip/linux/ignite3db [hunk 2]
```diff
@@ -59,7 +59,19 @@ start() {
...
+
+  max_attempts=10
+  count=0
+  while [ps -p "$(cat ${IGNITE_HOME}/pid)" > /dev/null]; do
```
Comment (Lozmanov) [inline packaging/db/zip/linux/ignite3db:66]: space between [ and ]

File: packaging/db/zip/linux/ignite3db [hunk 3]
```diff
@@ -59,7 +59,19 @@ start() {
 }
 
 stop() {
-  kill -9 "$(cat ${IGNITE_HOME}/pid)"
+  kill -15 "$(cat ${IGNITE_HOME}/pid)"
+
+  max_attempts=10
```
Comment (Lozmanov) [inline packaging/db/zip/linux/ignite3db:65]: unused

File: packaging/db/zip/linux/ignite3db [hunk 4]
```diff
@@ -59,7 +59,19 @@ start() {
...
+  count=0
+  while [ps -p "$(cat ${IGNITE_HOME}/pid)" > /dev/null]; do
+    sleep 10
+    count=$((count + 1))
+  done
+
+  if [ ps -p "$(cat ${IGNITE_HOME}/pid)" > /dev/null ]; then
```
Comment (Lozmanov) [inline packaging/db/zip/linux/ignite3db]: Use 'if cmd; then ..' to check exit code, or 'if [[ $(cmd) == .. ]]' to check output

File: packaging/db/zip/linux/ignite3db [hunk 5]
```diff
@@ -59,7 +59,20 @@ start() {
 }
 
 stop() {
-  kill -9 "$(cat ${IGNITE_HOME}/pid)"
+  ignite_pid = "$(cat ${IGNITE_HOME}/pid)"
+  kill -15 "$ignite_pid"
```
Comment (Lozmanov) [inline packaging/db/zip/linux/ignite3db:63]: -15 is a default value for kill

File: packaging/db/zip/linux/ignite3db [hunk 6]
```diff
@@ -59,7 +59,20 @@ start() {
...
+
+  max_attempts=10
+  count=0
+  while [ ps -p "$ignite_pid" > /dev/null ] && [ "$count" -lt "$max_attempts" ]; do
+    sleep 10
+    count=$((count + 1))
+  done
+
+  if [ ps -p "$ignite_pid" > /dev/null ]; then
```
Comment (Lozmanov) [inline packaging/db/zip/linux/ignite3db:72]: no need to use  [ ] at all.
Just 

if ps -p "$ignite_pid" > /dev/null; then

fi

File: packaging/db/zip/linux/ignite3db [hunk 7]
```diff
@@ -59,7 +59,20 @@ start() {
 }
 
 stop() {
-  kill -9 "$(cat ${IGNITE_HOME}/pid)"
+  ignite_pid = "$(cat ${IGNITE_HOME}/pid)"
+  kill -15 "$ignite_pid"
+
+  max_attempts=10
+  count=0
+  while [ ps -p "$ignite_pid" > /dev/null ] && [ "$count" -lt "$max_attempts" ]; do
```
Comment (Lozmanov) [inline packaging/db/zip/linux/ignite3db:67]: without [ ] in cmd

`while ps -p "$ignite_pid" > /dev/null && [ "$count" -lt "$max_attempts" ]; do
  sleep 10
  count=$((count + 1))
done`

File: packaging/db/zip/linux/ignite3db [hunk 8]
```diff
@@ -59,7 +59,20 @@ start() {
 }
 
 stop() {
```
Comment (YuryYudin) [inline packaging/db/zip/linux/ignite3db:61]: What about printing notification about the fact that node has successfully been stopped (or otherwise)

**Pull request merged**